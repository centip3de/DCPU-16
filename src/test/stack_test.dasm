SET PUSH 0x10
SET PUSH 0x20
SET X POP
SET Y POP
