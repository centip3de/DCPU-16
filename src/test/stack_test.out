********* REG DUMP *********
A: 0x0
B: 0x0
C: 0x0
X: 0x20
Y: 0x10
Z: 0x0
I: 0x0
J: 0x0
PC: 0x5
O: 0x0
SP: 0xffff
